pr_testing/test_multiple_prs.sh

```diff
@@ -1438,6 +1449,18 @@ for x in REPORT_OPTS BUILD_EXTERNAL DO_DUPLICATE_CHECKS DO_DAS_QUERY DO_TESTS CM
   eval echo "$x=\\\"$(echo \$$x)\\\"" >> $WORKSPACE/job.env
 done
 
+if [ "${BUILD_ONLY}" = "true" ]; then
+  if ${ALL_OK} ; then
+    echo "+1" > comment.txt
+  else
+    echo "-1" > comment.txt
+  fi
+  echo "" >> comment.txt
+  curl -L ${PR_COMMENT_TEXT_URL} >> comment.txt
+  $WORKSPACE/cms-bot/comment-gh-pr.py --repository ${PR_REPO} --pullrequest ${PR_NUMBER} --report-file comment.txt
+  mark_commit_status_all_prs "${PR_COMMIT_STATUS}" 'success' -u "${BUILD_URL}" -d "Finished" -e
```

Comment:
@iarspider , as discussed , based on test results it should be either `success` or `error`

Reply:
Notice that for regular tests we always set this commit status to success: https://github.com/cms-sw/cms-bot/blob/4f1278ad49b3215a14b577b88c0ff27334365b5c/process_pr.py#L2233-L2238

Reply:
ah ok, this is the one with `/optional` or `/required` . Yes that should be set to `success`